x/msgfees/spec/04_messages.md

```diff
@@ -0,0 +1,51 @@
+<!--
```

Comment:
This file should be called `xx_queries.md` since it's about the queries. The `xx_messages.md` docs are about the transaction messages.